code/controllers/subsystem/shuttle.dm

```diff
@@ -270,12 +270,16 @@ SUBSYSTEM_DEF(shuttle)
 	src.emergency = src.backup_shuttle
 
 /datum/controller/subsystem/shuttle/proc/cancelEvac(mob/user)
-	if(canRecall())
-		emergency.cancel(get_area(user))
-		log_game("[key_name(user)] has recalled the shuttle.")
-		message_admins("[ADMIN_LOOKUPFLW(user)] has recalled the shuttle.")
-		deadchat_broadcast(" has recalled the shuttle from <span class='name'>[get_area_name(user, TRUE)]</span>.", "<span class='name'>[user.real_name]</span>", user)
-		return 1
+	if(!canRecall())
+		return FALSE
+	if(isAI(user))
+		minor_announce("The shuttle has been recalled at <span class='name'>[get_area_name(user, TRUE)]</span>. Have a secure day.")
```

Comment:
this should be handled by the same mechanic that comms consoles use to find rogue consoles. No need to broadcast AI's area EVERY time they cancel the shuttle.

that `emergency.cancel` might be good place to start.